docs/tools/vdb_table/data/rockset.json

```diff
@@ -83,28 +83,28 @@
     "comment": ""
   },
   "rag": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/sofia099/vector_search_demo",
     "comment": ""
   },
   "recsys": {
     "support": "full",
-    "source_url": "",
+    "source_url": "https://github.com/rocksetlabs/realtime-semantic-search",
```

Comment:
This column is for whether the VDB provides a recommendations API. Example: https://qdrant.tech/documentation/concepts/search/#recommendation-api.

Please change to "none" if there isn't an equivalent.